cmd/cloud/setup.go

```diff
@@ -177,6 +174,10 @@ func checkToken(client astro.Client, coreClient astrocore.CoreClient, out io.Wri
 		if err != nil {
 			return err
 		}
+		err = c.SetContextKey("Organization", c.Organization)
```

Comment:
```suggestion
		err = c.SetContextKey("organization", c.Organization)
```
nit: I think all our keys are lower cases

Reply:
Oops, yes